pkg/preparer/orchestrate.go

```diff
@@ -418,7 +420,9 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 			}
 		} else {
 			// TODO: do this in a transaction
-			err = p.podStore.WriteRealityIndex(pair.PodUniqueKey, p.node)
+			// TODO dai - what do I do with the errors?
```

Comment:
Is it worth checking for errors on `transaction.Add()` if we know it won't error? What should the error handler do in that case?